src/platform/common/scriptConstants.ts

```diff
@@ -14,11 +14,6 @@ export namespace DataFrameLoading {
 
     export const DataFrameInfoFunc = '_VSCODE_getDataFrameInfo';
     export const DataFrameRowFunc = '_VSCODE_getDataFrameRows';
-
-    // Constants for the debugger which imports the script files
-    export const DataFrameImport = `__import__('vscodeDataFrame')`;
```

Comment:
These don't make sense anymore because we don't actually import a file. We can just reference the function name directly now.